libcudacxx/test/libcudacxx/std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/eof.pass.cpp

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
I would rather mark that as 
```suggestion

// UNSUPPORTED: true

```